backend/models/appModels/Tax.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+
+const taxSchema = new mongoose.Schema({
+  taxName: {
```

Comment:
removed: {
    type: boolean,
    default: false,
  },
 enabled: {
    type: boolean,
    default: true,
  },

Reply:
As I understand you want to add this two items in the model

Reply:
As I understand you want to add this two items in the model

Reply:
As I understand you want to add these two items to the model